sdks/python/apache_beam/runners/interactive/utils.py

```diff
@@ -32,13 +32,22 @@
 import apache_beam as beam
 from apache_beam.dataframe.convert import to_pcollection
 from apache_beam.dataframe.frame_base import DeferredBase
+from apache_beam.internal.gcp import auth
+from apache_beam.internal.http_client import get_new_http
+from apache_beam.io.gcp.internal.clients import storage
 from apache_beam.portability.api.beam_runner_api_pb2 import TestStreamPayload
 from apache_beam.runners.interactive.caching.cacheable import Cacheable
 from apache_beam.runners.interactive.caching.cacheable import CacheKey
 from apache_beam.runners.interactive.caching.expression_cache import ExpressionCache
 from apache_beam.testing.test_stream import WindowedValueHolder
 from apache_beam.typehints.schemas import named_fields_from_element_type
 
+# Protect against environments where apitools library is not available.
+try:
+  from apitools.base.py.exceptions import HttpError
+except ImportError:
```

Comment:
This doesn't seem right. You should probably merge the import in the try-except block in the `assert_bucket_exists ` function and add an except block for `ImportError` and log a similar warning.

I'll put the suggestion in place below.

Reply:
That would make sense, thanks for the example. I've changed the code to this style now.